go.mod

```diff
@@ -67,3 +68,7 @@ replace mellium.im/sasl => github.com/mellium/sasl v0.2.1
 replace golang.org/x/sys => golang.org/x/sys v0.0.0-20200826173525-f9321e4c35a6
 
 replace github.com/onflow/flow-go/crypto => ./crypto
+
+
+// TODO: temp to test transactions templates for QC
```

Comment:
Can we tag a branch on `github.com/onflow/flow-core-contracts/lib/go/templates` instead so that this will be consistent across different machines?